frontend/src/components/Jobs.vue

```diff
@@ -106,26 +138,40 @@ export default {
     fetchData() {
       return fetchCctrayJson()
           .then(this.marshalData)
-          .then((data) => this.jobs = data)
+          .then((data) => {
+            this.visibleJobs = data.filter(j => !preferences.isJobHidden(j.name))
```

Comment:
We can have **jobs** as a state variable and then we can use **computed** to figure out **visible** and **hidden** jobs

Reply:
Changed accordingly